src/assets/css/global/_main.scss

```diff
@@ -665,3 +670,32 @@ km-cni-version {
 .km-option-divider.mat-divider {
   margin: 10px 0;
 }
+
+km-machine-deployment-list {
+  .mat-card {
+    .mat-card-header {
+      .mat-card-title {
+        i {
+          margin: 0 12px 0 0;
+        }
+
+        .km-quota-widget {
+          border: 1px solid #e0e3e5;
```

Comment:
Color/theme related styles should be added in `theme.scss` and please also use theme variables.